android/src/main/java/io/ably/flutter/plugin/push/FirebaseMessagingReceiver.java

```diff
@@ -80,6 +80,11 @@ private Boolean isApplicationInForeground(final Context context) {
     // This only shows processes for the current android app.
     final List<ActivityManager.RunningAppProcessInfo> appProcesses = activityManager.getRunningAppProcesses();
 
+    if (appProcesses == null) {
+      // appProcesses is null if no processes are running.
```

Comment:
This code block is pretty much self explanatory, I think you should be ok to remove this comment 😄

Reply:
I'd to go the other way: add more comments: 56eb5316672cd2fb5ec9d19355a32bb62a1f8299

`getRunningAppProcesses` returns null when no processes are running, not an emptyList.

Reply:
As you pointed out in your last comment, the doc for the function is quite clear
> Returns a list of application processes that are running on the device.
> Note: this method is only intended for debugging or building a user-facing process management UI.
> Returns:
> Returns a list of RunningAppProcessInfo records, or null if there are no running processes (it will not return an empty list). This list ordering is not specified.

. However I understand why you don't find it intuitive. So up to you